themes/src/main/resources/theme/base/admin/resources/templates/kc-component-config.html

```diff
@@ -33,7 +33,7 @@
         </div>
 
         <div class="col-md-6" data-ng-show="option.type == 'Script'">
-            <div ng-model="config[option.name][0]" placeholder="Enter your script..." ui-ace="{ useWrapMode: true, showGutter: true, theme:'github', mode: 'javascript'}">
+            <div ng-model="config[option.name]" placeholder="Enter your script..." ui-ace="{ useWrapMode: true, showGutter: true, theme:'github', mode: 'javascript'}">
```

Comment:
I found it ... i needed to propagate the default values of the configuration properties within the `AuthenticationConfigCreateCtrl` in `realm.js`